tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -97,7 +97,19 @@
 
   <mksrf_furban>
     <entry>
-      <data_filename>lnd/clm2/rawdata/mksrf_urban_0.05x0.05_simyr2000.c220127.nc</data_filename>
+      <data_filename>/glade/p/cgd/tss/people/oleson/Dynamic_Urban_Data/GaoOneil/PCTURB_GRID/Historical/urban_properties_GaoOneil_05deg_ThreeClass_2000_c20220910.nc</data_filename>
+      <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_3x3min_nomask_cdf5_c200129.nc</mesh_filename>
+    </entry>
+    <entry pft_years="1850">
+      <data_filename>/glade/p/cgd/tss/people/oleson/Dynamic_Urban_Data/GaoOneil/PCTURB_GRID/Historical/urban_properties_GaoOneil_05deg_ThreeClass_1850_c20220910.nc</data_filename>
+      <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_3x3min_nomask_cdf5_c200129.nc</mesh_filename>
+    </entry>
+    <entry pft_years="2000">
+      <data_filename>/glade/p/cgd/tss/people/oleson/Dynamic_Urban_Data/GaoOneil/PCTURB_GRID/Historical/urban_properties_GaoOneil_05deg_ThreeClass_2000_c20220910.nc</data_filename>
+      <mesh_filename>lnd/clm2/mappingdata/grids/UNSTRUCTgrid_3x3min_nomask_cdf5_c200129.nc</mesh_filename>
+    </entry>
```

Comment:
I'm a bit confused by this.  Above that, lines 99-102 have what you are suggesting.  I assume that takes care of any case where years 1850, 2000, or 2005 are not specified explicitly.  I assume that I have to explicitly call out years 1850, 2000, and 2005.  I was following the mksrf_fvegtyp example.  Let me know the best way to do this, thanks.

Reply:
> Above that, lines 99-102 have what you are suggesting. I assume that takes care of any case where years 1850, 2000, or 2005 are not specified explicitly.

That's right. Lines 99-102 ensure that it will use 2000 for any year setting that's not 1850 and 2005.

The mksrf_fvegtyp example is similar, but it sets every case explicitly without a setting for cases that have not been called out.

Reply:
I get your point now.  I've removed the redundant call-out for year 2000.